adulbrich/osu-fsf
# Homepage refactor

Moved some of the functions in /events/+page.svelte to +page.server.ts.  

To test
- Navigate to src-web
- Run `npm run dev`
- Check if all of the links on the homepage work as intended
